.github/workflows/cypress.yml

```
@@ -0,0 +1,108 @@
+name: Running Cypress Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)